boards/rapidsilicon/virgo_tc_eval_soc_proto/Kconfig.virgo_tc_eval_soc_proto

```
@@ -0,0 +1,5 @@
+# Copyright (c) 2021 Andes Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_VIRGO_TC_EVAL_SOC_PROTO
+	select SOC_VIRGO if BOARD_VIRGO_TC_EVAL_SOC_PROTO
```

```suggestion
	select SOC_VIRGO
```
if condition is not required. If condition only required when this is dependent on some other condition other than BOARD_VIRGO_TC_EVAL_SOC_PROTO

---

Yes indeed. This was a copy past.